python/src/main/api.py

```diff
@@ -51,9 +63,35 @@ def get(self, email):
             return jsonify({'person': create_person(person).serialize()})
 
 
-# @api.route('/api/v1/items/<string:email>')
-# class Item(Resource):
-#     def put(self, ):
+@api.route('/api/v1/items/<string:person_email>/<string:name>/<string:price>/<int:priority>/<string:hyperlink>')
+class Items(Resource):
+     def put(self, person_email, name, price, priority, hyperlink):
+        conn = sqlite3.connect("person_and_item.db")
+        cursor = conn.cursor()
+        db = DB(cursor, conn)
+        item = Item(person_email, name, price, priority, hyperlink)
+        db.insert_item(item)
+        logging.info("Put item '%s'" % name)
+        conn.close()
+        return 200
+
+@api.route('/api/v1/items/<string:person_email>/<string:name>')
+class Itemz(Resource):
+     def delete(self, person_email, name):
+         conn = sqlite3.connect("person_and_item.db")
+         cursor = conn.cursor()
+         db = DB(cursor, conn)
+         items = db.get_specific_item(person_email, name)
```

Comment:
why not just get_item?
What does `specific` add here?

Reply:
if DC, Jake and Mander all want a Bike, then get item brings back all of them. get specific item would get back, for example, just Mander's

Reply:
that still is not a specific item lol
get_item_by_person then.